lib/gooddata/bricks/middleware/dwh_middleware.rb

```diff
@@ -13,6 +13,11 @@ module Bricks
     # Connects to the warehouse (ADS) and enriches parameters with GoodData::Datawarehouse
     class WarehouseMiddleware < Bricks::Middleware
       def call(params)
+        if ENV['RSPEC_ENV'] == 'test' && GoodData::Environment::VCR_ON
```

Comment:
It would be cleaner not to mix test logic with production code. How about monkey-patching `WarehouseMiddleware.call` in VCRConfigurer? That way all VCR-related logic would be together in one file.